website/content/en/docs/contribution-guidelines/developer-guide.md

```diff
@@ -1,97 +1,62 @@
-
-**NOTE** The Deploy stage (configured in [.travis.yml ][travis]) builds also execute the same [makefile][makefile] targets. This stage is executed against the master branch when a Pull Request is merged.
-
-## How to test the build of operator-sdk binaries
-
-Follow these steps to execute the Travis `Deploy` stage against your branch to demonstrate that the merge build will complete as expected.
+See the [testing][dev-testing] and [documentation][dev-docs] guides for more information.
 
-- Enable the Travis in your fork repository. For more info see [`To get started with Travis CI using GitHub`](https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci-using-github) 
-- Create image repos in quay (or another registry that supports multi-arch images) for ansible, helm, and scorecard proxy. For each image type, you need repos for the manifest list and one for each architecture (e.g. `ansible-operator`, `ansible-operator-amd64`, `ansible-operator-s390x`, etc.)
+## Releasing
 
-**NOTE** Be sure to make each repository public.
+See the [release guide][dev-release] for more information.
 
-- Set the following environment variables in the Travis settings for your fork:
+## Continuous Integration (CI)
 
-    - `ANSIBLE_IMAGE` docker image name (e.g. `quay.io/joelanford/ansible-operator`)
-    - `HELM_IMAGE` same as above, but for helm
-    - `SCORECARD_PROXY_IMAGE` same as above, but for scorecard proxy
-    - `DOCKER_USERNAME` credentials for your repo
-    - `DOCKER_PASSWORD` credentials for your repo
-    - `DOCKER_CLI_EXPERIMENTAL`  set to `enabled` 
-    - `COVERALLS_TOKEN`  token to integrate the project with `https://coveralls.io/`. So, enable your fork in `https://coveralls.io/` and generate a token to allow it. 
+The operator-sdk repo uses [Travis CI][travis] to test each pull request and build images for both master commits
+and releases. You can alter these processes by modifying this [`.travis.yml`][travis.yml] config file.
 
-- Make a commit with `[travis deploy]` in the commit message on the branch with the changes.
-- Check the travis build for your branch in your fork (not the PR build in the operator-sdk repo, since we don't allow PRs to build images in the `operator-framework` quay repo.)
+### Testing builds with new architectures
 
-**NOTE** Post a link in the Pull Request to the Travis build page showing successful `Deploy` and `Deploy multi-arch manifest lists` stages with your changes.
+Follow these steps to execute the Travis `Deploy` stage against your branch
+to demonstrate that the merge build will complete as expected.
 
-See the project [README][sdk-readme] for more details.
+- Enable Travis in your fork repository. See [this guide][travis-setup] for more information.
+- Create _public_ image repos in a registry that supports multi-arch images for each image built by `make image-build`,
```

Comment:
this wording is a bit awkward. Maybe 'Create a public image repo for each architecture built by `make image-build`. Make sure your registry supports multiple image architectures, such as quay.io'? Also not sure about the italics for the word public?

Reply:
Reworded this.